test/python/common.py

```diff
@@ -193,7 +193,7 @@ def wait_for_initialization(self, job, timeout=1):
         status."""
         waited = 0
         wait = 0.1
-        while job.status['status'] == JobStatus.INITIALIZING:
+        while job.status()['status'] == JobStatus.INITIALIZING:
```

Comment:
Can you double-check this line (and the analogous one in `assertStatus`)? If `job.status()` is returning an enum, the access to `['status']` would fail.